src/CommandingOfficers/Meridian.java

```diff
@@ -0,0 +1,375 @@
+package CommandingOfficers;
+
+import java.util.ArrayList;
+
+import CommandingOfficers.Modifiers.CODamageModifier;
+import CommandingOfficers.Modifiers.CODefenseModifier;
+import Engine.GameAction;
+import Engine.GameActionSet;
+import Engine.Path;
+import Engine.UnitActionType;
+import Engine.Utils;
+import Engine.XYCoord;
+import Engine.Combat.BattleInstance.BattleParams;
+import Engine.GameEvents.GameEvent;
+import Engine.GameEvents.GameEventListener;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameInput.GameInputState;
+import Engine.GameInput.GameInputState.StateData;
+import Terrain.GameMap;
+import Terrain.Location;
+import Terrain.MapMaster;
+import UI.MapView;
+import UI.Art.Animation.GameAnimation;
+import Units.Unit;
+import Units.UnitModel;
+import Units.Weapons.Weapon;
+import Units.Weapons.WeaponModel;
+
+/**
+ * Commander Meridian focuses on a unique brand of flexibility; she transforms her tanks and artillery into each other 
+ */
+public class Meridian extends Commander
+{
+  private static final CommanderInfo coInfo = new instantiator();
+  private static class instantiator extends CommanderInfo
+  {
+    public instantiator()
+    {
+      super("Meridian");
+      infoPages.add(new InfoPage(
+          "While other commanders may focus on fine-grain tactics, Meridian has taken a different line: her basic tanks and artillery are built from the same parts, and can be re-configured on the field at no cost.\n" + 
```

Comment:
"fine-grained". Also, "line" feels like it should be "tack"

Reply:
The tactics are not fine-grained, they are *along* the fine grain. I wouldn't describe Max as using "fine-grained" tactics, but his tactics are better than others' when in a fine-grain analysis i.e. on the level of individual tanks in a unit.

"Line" is a pun, I'm not changing it.

Reply:
uhhhh, ok. I guess. "Unit-level" might be more clear than "fine-grain" though.

Reply:
"Unit-level" feels too game-y IMO.
Could just say "micro tactics" or "micro-level tactics"?

Reply:
I mean. This is a game. But: "granular", "small-scale", "individual", "Lilliputian-esque"
I'm not sure that transforming units counts as the opposite of all that, but eh.

Reply:
I like "granular".
Jess's tanks and artillery are a little better at killing things than average.
Meridian's tank can turn into an artillery to be effective vs a Md. Tank, and her arty can become a tank to win against an opposing arty.

The former seems more like a tactical benefit, and the latter a strategic benefit.
Tactical advantages can create strategic advantages, but they're not exactly the same.